front/app/containers/Admin/projects/project/nativeSurvey/FormResults/AnalysisBanner.tsx

```diff
@@ -55,27 +50,45 @@ const CreateAnalysisModal = ({ onClose }: { onClose: () => void }) => {
     );
   };
 
+  if (isNilOrError(formCustomFields)) return null;
+
   return (
     <Box px="48px">
-      <Title mb="48px">{formatMessage(messages.analysisSelectQuestions)}</Title>
-      {textCustomFields?.map((field) => {
-        return (
-          <ListItem key={field.id} py="16px">
-            <Checkbox
-              label={localize(field.title_multiloc)}
-              checked={selectdQuestions.includes(field.id)}
-              onChange={(e) => {
-                if (e.target.checked) {
-                  setSelectedQuestions([...selectdQuestions, field.id]);
-                } else {
-                  setSelectedQuestions(
-                    selectdQuestions.filter((id) => id !== field.id)
-                  );
-                }
-              }}
-            />
-          </ListItem>
-        );
+      <Title>{formatMessage(messages.analysisSelectQuestions)}</Title>
+      <Text>
+        Which questions do you want to analyze simultaneously? You can always
+        create a new analysis with different questions later.
+      </Text>
+      {formCustomFields?.map((field) => {
+        if (field.input_type === 'page') {
+          return (
+            <Title variant="h5" key={field.id}>
+              <T value={field.title_multiloc} />
```

Comment:
We are already using `useLocalize` in this component which is serving the same purpose as `T`. Minor but perhaps it's better to choose one and use it consistently within the same component.